nibabel/gifti/tests/data/ascii.gii

```diff
@@ -48,15 +48,15 @@
          <DataSpace><![CDATA[NIFTI_XFORM_TALAIRACH]]></DataSpace>
          <TransformedSpace><![CDATA[NIFTI_XFORM_TALAIRACH]]></TransformedSpace>
          <MatrixData>
-            1.000000 0.000000 0.000000 0.000000 
-            0.000000 1.000000 0.000000 0.000000 
-            0.000000 0.000000 1.000000 0.000000 
-            0.000000 0.000000 0.000000 1.000000 
+            1.000000 0.000000 0.000000 0.000000
+            0.000000 1.000000 0.000000 0.000000
+            0.000000 0.000000 1.000000 0.000000
+            0.000000 0.000000 0.000000 1.000000
```

Comment:
While some of these are probably semantically null changes, I think it's bad policy to automatically change a data file. We should cordon off `data/` directories from being inspected/modified by linters.

Reply:
Reverted changes to data directories and added `'.*/data/.*'` to the pre-commit configuration top-level exclusion regex :+1: